pkg/services/network/constants.go

```diff
@@ -30,9 +30,4 @@ const (
 
 	// SystemNamespace is the namespace where supervisor control plane VMs reside.
 	SystemNamespace = "kube-system"
-
-	// legacyDefaultNetworkLabel was the label used for default networks.
-	//
-	// Deprecated: legacyDefaultNetworkLabel will be removed in a future release.
-	legacyDefaultNetworkLabel = "capw.vmware.com/is-default-network"
```

Comment:
We have to keep these as part of https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/2394

Current plan is to not move forward on that issue until we have more clarity from the CAPW folks.

Reply:
Done